08-confidence-intervals.Rmd

```diff
@@ -338,36 +367,36 @@ Let's now mimic our tactile resampling activity virtually with a computer.
 
 ### Virtually resampling once
 
-First, let's perform the virtual analog of resampling once. Recall that the `pennies_sample` data frame included in the `moderndive` package contains the years of our original sample of 50 pennies from the bank. Furthermore, recall in Chapter \@ref(sampling) on sampling that we used the `rep_sample_n()` function as a virtual shovel to sample balls from our virtual bowl of `r nrow(bowl)` balls as follows: 
+First, let's perform the virtual analog of resampling once. Recall that the `pennies_sample` data frame included in the `moderndive` package contains the years of our original sample of `r num_pennies` pennies from the bank. Furthermore, recall in Chapter \@ref(sampling) on sampling that we used the `rep_sample_n()` function as a virtual shovel to sample balls from our virtual bowl of `r nrow(bowl)` balls as follows: 
 
 ```{r, eval=FALSE}
 virtual_shovel <- bowl %>% 
   rep_sample_n(size = 50)
```

Comment:
Not sure if there is a way to also use `num_pennies` here instead of 50, but it would be really cool to see if that's an option as well.

Reply:
I can't find a way to do this but it really would be cool to see it happen!